Jropp/jason-ropp-js-iobp
# Polymer Upgrade: #4 Expand collapse component

## What It Does

Moves expand/collapse clickable on existing user card into it's own custom component.

This simplifies readability and maintainability of code by moving styling and toggle of jha-arrow-icons into its own component.

Also explain _how_ it was implemented, in general terms.

## How To Test

Click on the expand details button. It should expand properly and the arrow should change to reflect that it is now a collapse details button.

## Checklist

Under penalty of public shaming, I avow that I:


Browsers tested:


## Dependencies

- [ ] #172 
- [ ] #173 
- [ ] #170
